src/coredump/coredump.c

```diff
@@ -1212,8 +1218,25 @@ static int process_kernel(int argc, char* argv[]) {
 
         if (context.is_journald || context.is_pid1)
                 r = submit_coredump(&context, iovw, STDIN_FILENO);
-        else
+        else {
+                _cleanup_free_ char *process_name;
+                process_name = NULL;
+                process_name = filename_escape(context.meta[META_COMM]);
+                if (!process_name)
+                        log_error_errno(r, "Failed to escape process name. Ignoring coredump ratelimit: %m");
```

Comment:
r is not set here.

also, this can only fail on OOm, hence maybe just call log_oom() here

Reply:
I have addressed the review comment. Please check once